math/benches/fft.rs

```diff
@@ -0,0 +1,108 @@
+use const_random::const_random;
+use criterion::{black_box, criterion_group, criterion_main, Criterion};
+use lambdaworks_math::fft::fft_cooley_tukey::{fft, inverse_fft};
+use lambdaworks_math::field::element::FieldElement;
+use lambdaworks_math::field::traits::{IsField, IsTwoAdicField};
+use rand::Rng;
+
+// Copied from math/src/field/test_fields/u64_test_field.rs
+#[derive(Debug, Clone, PartialEq, Eq)]
+struct U64TestField<const MODULUS: u64>;
```

Comment:
This was needed because the one defined in `u64_test_field.rs` is not exported outside the crate, should we export it instead to use here?

Reply:
We can make it Pub, since its under test fields and people will not confuse it with common fields, as to avoid the duplicated code.

Reply:
Made it public